dust_extinction/dust_extinction.py

```diff
@@ -1200,22 +1198,24 @@ def evaluate(self, in_x, Rv):
 
         # terms depending on Rv
         C2 = -0.824 + 4.717/Rv
-        # updated for FM07 correlation between C1 and C2
+        # original F99 C1-C2 correlation
         C1 = 2.030 - 3.007*C2
```

Comment:
This did not change from the previous.  So was the previous really the F99 version or the FM07 version?

Reply:
It is the original F99.  The newer FM07 is C1 = 2.09 - 2.84 C2.

It does gets confusing.  Here is a cheat sheet I've been making for myself to keep it straight:

![image](https://user-images.githubusercontent.com/6710382/36687790-74be30cc-1adf-11e8-9850-2e89e88e2b77.png)